Doc/library/ast.rst

```diff
@@ -139,9 +139,10 @@ Literals
 
    .. doctest::
 
+        >>> import ast
```

Comment:
This should be necessary for the doctest as we have

```
   .. testsetup::
    
        import ast
```

at the beggining of the document. Maybe I am missing something, though

Reply:
I am not experienced with doctests. I ran `./python -m doctest Doc/library/ast.rst` and it failed. Seems `doctest` does not understand `testsetup`. Indeed, it is a Sphinx extension.

Reply:
Yeah, you need to do "make doctest" or `PATH=./venv/bin:$PATH sphinx-build -b doctest -d build/doctrees   -W . build/doctest FILENAME`  if you want to only test one single file (the first run will be a bit longer while it builds the tree).